CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ elseif(${CMAKE_Fortran_COMPILER_ID} MATCHES "GNU")
 elseif(${CMAKE_Fortran_COMPILER_ID} MATCHES "PGI")
   list(APPEND musica_compile_definitions MUSICA_USING_PGI)
 endif()
+set(CMAKE_Fortran_FLAGS "-fPIE")
```

Comment:
if we need to set this, then please set it with `[target_compile_options](https://cmake.org/cmake/help/latest/command/target_compile_options.html)` on the appropriate target (musica fortran?). This prevents us from polluting downstream builds with options that they may not want.

Reply:
I've taken this out for now ... I don't understand why I needed it on one Linux OS (RedHat) with GCC 13.2 but not on another (Ubuntu) with GCC 11.4.